simul/platform/.gitignore

```diff
@@ -1,2 +0,0 @@
-mininet/
```

Comment:
this used to be a binary-folder, now it's the `start.py` and `setup*.sh`-folder.